### PR TITLE
FUSETOOLS2-709 - refactor test to have more information on failure

### PR DESCRIPTION
**Pull request by apupier**
I created https://issues.redhat.com/browse/FUSETOOLS2-713 to try to avoid breaking the master on each Camel K release.

we can notice that the test is failing with a more precise message here https://travis-ci.org/github/camel-tooling/vscode-camelk/builds/729248437#L711 This is checked via separate temporary branch/[PR](https://github.com/camel-tooling/vscode-camelk/pull/555) because the test requires to have access to github token.

### CODE REVIEW

File: src/test/suite/install.test.ts [hunk 1]
```diff
@@ -16,7 +16,7 @@
  */
 'use strict';
 
-import * as assert from 'assert';
+import {assert} from 'chai';
```
Comment (apupier) [inline src/test/suite/install.test.ts:19]: it provides more functionality and avoids usage of deprecated "assert.equal" method

**Review by lhein — DISMISSED** [review]

**Comment by lhein** [comment]
good idea

File: src/test/suite/install.test.ts [hunk 2]
```diff
@@ -54,29 +54,26 @@ suite("ensure install methods are functioning as expected", function() {
...
+		skipIfNoGithubTokenAvailable(testInvalidVersionVar);
 
-		await versionUtils.checkKamelNeedsUpdate('bogusversion').then( () => {
-			assert.fail('cli version checking, negative case worked');
-		}).catch( (error) => {
-			assert.ok(error, 'cli version checking, negative case worked');
-		});
+		const needsUpdate: boolean = await versionUtils.checkKamelNeedsUpdate('bogusversion');
+		assert.isTrue(needsUpdate, 'It is mentioned that there is no need of update even with an invalid kamel version');
+	});
```
Comment (apupier) [inline src/test/suite/install.test.ts:62]: argh, the test wasn't checking that. the checkKamelNeedsUpdate seems to not care of the version provided https://travis-ci.org/github/camel-tooling/vscode-camelk/builds/729244531#L706-L711

Comment (apupier) [reply src/test/suite/install.test.ts]: fixed:
- without update of Camel K defautl version: https://travis-ci.org/github/camel-tooling/vscode-camelk/builds/729248437#L603 
- with update of Camel K default version https://travis-ci.org/github/camel-tooling/vscode-camelk/builds/729248668#L607

**Review by lhein — APPROVED** [review]

**Review by bfitzpat — APPROVED** [review]

**Pull request merged**